pkg/asset/agent/image/agentimage.go

```diff
@@ -131,6 +135,44 @@ func (a *AgentImage) updateIgnitionImg(ignition []byte) error {
 	return nil
 }
 
+// updateIgnitionInfo updates igninfo.json with the current size of
+// the ignition buffer.
+func (a *AgentImage) updateIgnitionInfo(ignitionSize int) error {
+	ignInfoPath := filepath.Join(a.tmpPath, "coreos", "igninfo.json")
+	ignInfoRaw, err := os.ReadFile(ignInfoPath)
+	if err != nil {
+		// Skip the update if the file does not exist
+		if os.IsNotExist(err) {
```

Comment:
Seems this should be logged, otherwise there will just be a confusing log msg in the calling function that the image size was increased.

Reply:
I'm not sure about what could be the confusing message: increasing the ignition image is independent by the presence of the `igninfo.json` file or not. If there is, it means it's a newer ISO - and thus it must be updated. If there isn't, it means it's an older ISO (and in that case the coreos-installer will simply just look at the file size)

Reply:
OK, I guess I didn't get that the message re: `ignition content length exceeds default embed area size increasing final image size` would still be relevant if the ignition info wasn't updated.